Dockerfile

```diff
@@ -0,0 +1,30 @@
+#Build: docker build -t jest .
+#Run: docker run --volume="$PWD:/usr/src/app" --rm jest:latest
+
+#Use Python 2.7.17 image
+FROM python:2.7.17
```

Comment:
this should use the node images, which comes with yarn already

Reply:
When I try to use `node:12.13.0-slim`, with the following Dockerfile, I get this error log:

![Untitled](https://user-images.githubusercontent.com/28692645/67972861-1a569b00-fbd5-11e9-9210-fd87b011405f.png)

From what I understand, Jest requires Python 2.x? Is this included in a `NodeJS` image?

If not, I thought it'd be easier to use the python image, then install NodeJS.

Reply:
We need a bunch of stuff. We should use the node image and install (at least) python and mercurial. We might need more stuff like cmake or gcc - would need to build the image to see

Reply:
@SimenB thanks for the feedback. I'll refactor the image to use node and install python and mercurial. I'll let you know if I run into any roadblocks.

Reply:
@SimenB I pushed a new commit that has this working. Looks like `node:12.13.0` includes `python2.7` so there was no need to setup Python.

My thoughts are that this image is for contributing to the Jest "core", not the website. We should have a separate issue / pr for Dockerizing the website (which I'd be happy to do).